manifest.yml

```diff
@@ -0,0 +1,22 @@
+---
+applications:
+- name: govuk-dev-content-store
+  memory: 256M
+  buildpack: ruby_buildpack
+  instances: 1
+  services:
+  - content-store-mongo
+  env:
+    GOVUK_APP_NAME: content-store
+    GOVUK_CONTENT_SCHEMAS_PATH: /govuk-content-schemas
+    # I had to add these. I made up the values, no idea what they *should* be
```

Comment:
These appear to be set here at the moment: https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/deploy/config.pp

For the vagrant dev environment, app domain is "dev.gov.uk" and website root is  "http://www.dev.gov.uk".

It's probably not actually used for content store, but to be consistent with other apps it should use the environment specific domain.